app/src/main/java/waiter/App.java

```diff
@@ -9,20 +9,54 @@
 import waiter.Threadable.ThreadGenerator;
 import waiter.Transportable.Messenger;
 
+import java.util.concurrent.Callable;
+
 class App {
 
     public static void main(String[] args) {
 
         int port = 5000;
         Listener listener = new Listener();
+        Routes routes = constructRoutes();
         HttProtocol protocol = new HttProtocol(
                 new RequestParser(),
-                new Router(),
-                new ResponseFormatter()
+                new Router(routes)
         );
         Messenger messenger = new Messenger(protocol);
         ThreadGenerator threadGenerator = new ThreadGenerator();
         EchoServer echoServer = new EchoServer(new Communicator(threadGenerator, listener, messenger));
         echoServer.start(port);
     }
+
+    private static Routes constructRoutes() {
+        Routes routes = new Routes();
+        RouteBuilder routeBuilder = new RouteBuilder();
+
+        routes.addRoute(
+                routeBuilder.newUp()
+                        .url("/simple_get")
+                        .methods(new String[]{"GET", "HEAD"})
+                        .handler(okHandler)
+                        .build()
+        );
+
+        routes.addRoute(
+                routeBuilder.newUp()
+                        .url("/simple_get_with_body")
+                        .methods(new String[]{"GET"})
```

Comment:
I think I used the Enums in [this commit](https://github.com/jonmanahan/waiter/pull/18/commits/c961b693c2652acfc53b02518629f441292d8b50), but I don't think Im using them to the best of their abilities since I am immediately converting them to strings, Ill look into making the utilize Enums throughout!

Reply:
Im gonna look to separate PR this if thats all good with you, if not no worries, just figured this was on the lower end of priority, either way let me know 😄

Reply:
The idea of using Enum is that status code and method, even though they are represented as strings to us, have specific formats and values. For example `200 OK` instead of `20 ok`, `GET` instead of `gET` or `get`. By using string we're saying, as long as they are string, they'll work as described in the HTTP specifications but that's not the case. Enum combined with Java's type safety helps us define that format and value so that there's no confusion as to what those requirements are. Also imagine if the specification changes, we only need to update one spot.

The purpose of Enum is essentially to make devs' life easier, the same with the builder's pattern. We could bypass all of that and construct a response in a string but that'll be tedious and error-prone.